decred/decred/dcr/txscript.py

```diff
@@ -1814,6 +1818,10 @@ def payToStakePKHScript(addr, stakeCode):
     Returns:
         ByteArray: The script that pays to the pubkey hash of the address.
     """
+    if not isinstance(addr, addrlib.AddressPubKeyHash):
+        raise DecredError(f"address is not an AddressPubKeyHash")
+    if stakeCode not in (opcode.OP_SSTX, opcode.OP_SSTXCHANGE):
+        raise DecredError(f"stake code is not sstx or sstxchange: {hex(stakeCode)}")
```

Comment:
Why not `OP_SSRTX`? In fact, now that I'm looking at it, why doesn't `payToSSRtxPKHDirect` just use `payToStakePKHScript` internally?